src/index.js

```diff
@@ -0,0 +1,70 @@
+function g(e) {
+  return document.getElementById(e);
+}
+function refresh(gameID) {
```

Comment:
- [ ] As mentioned in the project requirements, kindly use the arrow functions instead of the function keyword since we're working and practicing the ES6.
_Kindly apply the changes to all similar situations._ 😊